README.md

```diff
@@ -30,7 +30,7 @@ Interact with your containers directly: pause, restart and stop containers. Laun
 ## <a name="getting-started"></a>Getting started
 
 ```
-sudo wget -O /usr/local/bin/scope https://git.io/scope
+sudo curl -L git.io/scope -o /usr/local/bin/scope
```

Comment:
I guess thats fine?  I mean, if you're downloading a shell script as root, https isn't exactly going to save you...